lib/docker/template/cache.rb

```diff
@@ -42,9 +42,10 @@ def aliased_context(builder)
       # --
 
       def cleanup(repo)
+        return unless repo.clean_cache?
         cache_dir = repo.cache_dir.parent
 
-        if repo.cacheable? && cache_dir.exist?
+        if cache_dir.exist?
```

Comment:
As I have tested `repo.cacheable?` will be false in the following case:

`docker-template clean`

When `Opts.yml` doesn't have a `cache: true` property. Therefore, the clean up will not be performed. As I see the following as a use scenario, this might be a "bug":

```shell
$ docker-template cache --no-clean
$ # some code that might use cache files.
$ docker-template clean # if Opts.yml doesn't have a cache: true, cache will not be clean up
```

This is specifically why I removed the expression.

That been said, I don't know about the negative impact it would have on Rootfs images. So, I ask you what would you think as a solution for both cases. There is a example of Rootfs image that I can use for tests and debug to help finding out a solution?